src/sparse_blas/backends/mkl_common/mkl_helper.hpp

```diff
@@ -20,6 +20,7 @@
 // MKLCPU and MKLGPU backends include
 // This include defines its own oneapi::mkl::sparse namespace with some of the types that are used here: matrix_handle_t, index_base, transpose, uolo, diag.
 #include <oneapi/mkl/spblas.hpp>
+#include "mkl_version.h"
```

Comment:
this doesn't hurt, but isn't strictly necessary now that we have removed the MACROs.  OK to keep ?

Reply:
It's best to remove.
```suggestion

```